Domains/4-Application/ISM/Released/IntegratedStructuralModel.02.00.00.ecschema.xml

```diff
@@ -0,0 +1,2755 @@
+        <ECEnumerator value="ismCurveChain2d" name="IsmCurveChain2d" displayLabel="Ism CurveChain 2d" />
+        <ECEnumerator value="ismPerimeterSurface2d" name="IsmPerimeterSurface2d" displayLabel="Ism PerimeterSurface 2d" />
+        <ECEnumerator value="ismPoint3dArray" name="IsmPoint3dArray" displayLabel="Ism Point 3d Array" />
+        <ECEnumerator value="ismMultiSurface2d" name="IsmMultiSurface2d" displayLabel="Ism Multi Surface 2d" />
+        <ECEnumerator value="ismTrimmedSurface3d" name="IsmTrimmedSurface3d" displayLabel="Ism Trimmed Surface 3d" />
+    </ECEnumeration>
+
+    <ECStructClass typeName="IsmPrimitiveGeometry" description="Primitive Geometry" modifier="Sealed">
+        <ECProperty propertyName="type" typeName="IsmGeometryType" description="Type of geometry." />
+        <ECProperty propertyName="geometry" typeName="Bentley.Geometry.Common.IGeometry" displayLabel="Geometry" kindOfQuantity="AECU:LENGTH" description="Actual geometry." />
+    </ECStructClass>
+
+    <ECStructClass typeName="IsmArrayGeometry" description="Array Geometry" modifier="Sealed">
+        <ECProperty propertyName="type" typeName="IsmGeometryType" description="Type of geometry." />
+        <ECArrayProperty propertyName="geometry" typeName="Bentley.Geometry.Common.IGeometry" displayLabel="Geometry" kindOfQuantity="AECU:LENGTH" description="An array of geometries representing more complex geometry." />
+    </ECStructClass>
+
+    <ECRelationshipClass typeName="IsmObject_Object" modifier="None" strength="referencing" strengthDirection="forward">
+        <BaseClass>bis:ElementRefersToElements</BaseClass>
+        <Source multiplicity="(0..*)" roleLabel="has IsmObject" polymorphic="true">
+            <Class class="bis:Element" />
+        </Source>
+        <Target multiplicity="(0..*)" roleLabel="is used by IsmObject" polymorphic="true">
+            <Class class="bis:Element" />
+        </Target>
+    </ECRelationshipClass>
+
+    <ECEntityClass typeName="IIsmPropertyHolder" modifier="Abstract">
+        <ECCustomAttributes>
+            <IsMixin xmlns="CoreCustomAttributes.01.00.03">
+                <AppliesToEntityClass>bis:Element</AppliesToEntityClass>
+            </IsMixin>
+        </ECCustomAttributes>
+    </ECEntityClass>
+
+    <PropertyCategory typeName="IIsmObject_category" displayLabel="Object" priority="200" />
+    <ECEntityClass typeName="IIsmObject" displayLabel="Ism Object" modifier="Abstract">
+        <BaseClass>IIsmPropertyHolder</BaseClass>
+        <ECCustomAttributes>
+            <IsMixin xmlns="CoreCustomAttributes.01.00.03">
+                <AppliesToEntityClass>bis:Element</AppliesToEntityClass>
+            </IsMixin>
+        </ECCustomAttributes>
+        <ECProperty propertyName="CustomProperties" typeName="string" displayLabel="Custom Property Values" category="IIsmObject_category" description="Custom properties." />
```

Comment:
Is this JSON?  If so please add extendedType attribute
```suggestion
        <ECProperty propertyName="CustomProperties" typeName="string" extendedTypeName="Json" displayLabel="Custom Property Values" category="IIsmObject_category" description="Custom properties." />
```

Reply:
OK, we can leave it is for now.  We could add behavior for a new xml extended type name but for now nothing is supported.  Technically any string is supported by EC, but we've been trying to use only supported names in BIS schemas.  I guess it's silly to make the schema less expressive just because we haven't standardized on a name yet.  @diegoalexdiaz  what do you think?  We could add 'xml' as a validated extended type in BIS schema validation easily.